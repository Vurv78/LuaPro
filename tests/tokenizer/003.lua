"Hello, \r world!"

[[Hello
World]]