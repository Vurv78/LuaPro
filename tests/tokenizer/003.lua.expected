local lupa = require("src.lupa")
local Token, Variant = lupa.Token, lupa.Token.Variant

return {
	Token.new(Variant.String, "Hello, \\r world!"),
	Token.new(Variant.MString, "Hello\nWorld"),
}